### PR TITLE
issue-25: pre check range support on server

### PR DESCRIPTION
**Pull request by setraj**
Issue link: https://github.com/arvyshka/blazer/issues/25

- Checked the "Accept-Ranges" header to check if the server supports the Range header in the request.
- Created String constant for user message

### CODE REVIEW

**Comment by arvyshka** [comment]
related: https://github.com/arvyshka/blazer/issues/25

File: internals/network/concurrent_downloader.go [hunk 1]
```diff
@@ -11,6 +11,9 @@ import (
 	pkg "github.com/arvyshka/blazer/pkg/data"
 )
 
+const (
```
Comment (arvyshka) [inline internals/network/concurrent_downloader.go:14]: is it better to keep this const in globals.go ?

Comment (setraj) [reply internals/network/concurrent_downloader.go:14]: I had the same thought then I looked for the best practices.
https://stackoverflow.com/a/6220792/4187022

**Comment by arvyshka** [comment]
@setraj : DoesServerSupportRangeHeader check has to happen much earlier , precisely before this line cmd/blazer.go#40  because the session-id (ID combination of url and thread count) is used as the folder name of the temp file that or CLI creates for storing segments. 

you might have not thought through this because the cmd/blazer.go is written very poorly (needs to be orgainzed bettter) , it is the starting point of our CLI

File: internals/network/concurrent_downloader.go (hunk 1 above)
Comment (arvyshka) [reply internals/network/concurrent_downloader.go:14]: we will create an issue to remove that global.go file too

**Comment by setraj** [comment]
we should set flags.Thread = 1, before cmd/blazer.go#40 so that folder name is consistent.
is that what you meant?

**Comment by arvyshka** [comment]
> we should set flags.Thread = 1, before cmd/blazer.go#40 so that folder name is consistent.
> is that what you meant?

yes exactly

**Review by arvyshka — APPROVED** [review]
LGTM

**Pull request merged**